quarkus.jdt.ext/com.redhat.microprofile.jdt.quarkus.test/src/main/java/com/redhat/microprofile/jdt/quarkus/QuarkusMavenProjectName.java

```diff
@@ -20,6 +20,6 @@ public class QuarkusMavenProjectName extends MicroProfileMavenProjectName {
 	public static String scheduler_quickstart = "scheduler-quickstart";
 	public static String scheduler_diagnostic = "scheduler-diagnostic";
 	public static final String quarkus_route = "quarkus-route";
-
+	public static final String clinit = "clinit";

```

Comment:
It seems that you don't use this constant, right?

Reply:
I use it here https://github.com/redhat-developer/quarkus-ls/pull/1059/files#diff-5a589c56b38b81bf238303c13413a59cd85109cb5bd81389b2980f7d41852db8R298